web/app.yaml

```diff
@@ -0,0 +1,9 @@
+runtime: nodejs10
+service: default
+
+handlers:
+  - url: /
+    static_files: build/index.html
+    upload: build/index.html
+  - url: /
```

Comment:
nit: add empty line above (following the example)

Reply:
@ifhuang What example are you referring to? I think it's abit odd to put a line break because they are both part of `handlers` though!

Reply:
https://cloud.google.com/appengine/docs/standard/python/config/appref#example

Reply:
Ohh okay I'll follow that then hahaha